src/applications/FlowrPhone/phoneWindow.ts

```diff
@@ -74,16 +74,19 @@ export class PhoneWindow extends KeyboardMixin(BrowserWindow) {
       },
       'update-phone-store': this.updateStore.bind(this),
         initProps: () => {
-          this.webContents.send( 'init-props',
+          const { username, host } =  props.registerProps
+          this._history = props.history
+          this._currentUser = props.currentUser
```

Comment:
I don't know if it's mandatory to assign the props, but it was like this before

Reply:
Yes it is 
<img width="319" alt="image" src="https://user-images.githubusercontent.com/5877409/144374717-9d919835-2b5f-4cb3-b314-9c01c170316d.png">